<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生成绩管理系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{border-image:url(:/mainWindow.jpg);}
#toolBar{background-image:url(:/mainWindow.jpg);}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="1">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>AcadEref</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>学生成绩管理系统</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>60</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="addPushButton">
        <property name="text">
         <string>增  加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deletePushButton">
        <property name="text">
         <string>删  除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="modifyPushButton">
        <property name="text">
         <string>修  改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="queryPushButton">
        <property name="text">
         <string>查  询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showPushButton">
        <property name="text">
         <string>显示</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="studentNumberLable">
          <property name="font">
           <font>
            <family>AcadEref</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>学 号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="studentNumberLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="studentNameLable">
          <property name="font">
           <font>
            <family>AcadEref</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>姓 名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="studentNameLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="studentClassLable">
          <property name="font">
           <font>
            <family>AcadEref</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>班 级</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="studentClassLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="studentScoreLable">
          <property name="font">
           <font>
            <family>AcadEref</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>成 绩</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="studentScoreLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="9" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>100</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="1">
     <widget class="QTextEdit" name="showTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionADD"/>
    <addaction name="actionDEL"/>
    <addaction name="actionMOD"/>
    <addaction name="actionSER"/>
    <addaction name="actionSHOW"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionADD"/>
   <addaction name="actionDEL"/>
   <addaction name="actionMOD"/>
   <addaction name="actionSER"/>
   <addaction name="actionSHOW"/>
  </widget>
  <action name="actionADD">
   <property name="icon">
    <iconset resource="ico/ico.qrc">
     <normaloff>:/add.png</normaloff>:/add.png</iconset>
   </property>
   <property name="text">
    <string notr="true">增加</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionDEL">
   <property name="icon">
    <iconset resource="ico/ico.qrc">
     <normaloff>:/del.png</normaloff>:/del.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionMOD">
   <property name="icon">
    <iconset resource="ico/ico.qrc">
     <normaloff>:/modify.png</normaloff>:/modify.png</iconset>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
   <property name="toolTip">
    <string>修改</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionSER">
   <property name="icon">
    <iconset resource="ico/ico.qrc">
     <normaloff>:/search.png</normaloff>:/search.png</iconset>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
   <property name="toolTip">
    <string>查询</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSHOW">
   <property name="icon">
    <iconset resource="ico/ico.qrc">
     <normaloff>:/show.png</normaloff>:/show.png</iconset>
   </property>
   <property name="text">
    <string>显示所有</string>
   </property>
   <property name="toolTip">
    <string>显示所有</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ico/ico.qrc"/>
 </resources>
 <connections/>
</ui>
